content/en/blog/_posts/2022-12-06-kubernetes-1.25-blog.md

```diff
@@ -0,0 +1,205 @@
+
+This release will be the first one that will be exclusively published in the new [registry.k8s.io](http://registry.k8s.io) image registry: no tags from v1.26 will be published, and only tags from releases before v1.25 will continue to be updated, in the (now legacy) k8s.gcr.io image registry. Refer to [this recently published article](/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/) for more information on the motivation, advantages, and implications of this significant change.
+
+### CRI v1alpha2 removed
+
+With the adoption of the [Container Runtime Interface](https://kubernetes.io/docs/concepts/architecture/cri/) (CRI) and the [removal of dockershim](https://kubernetes.io/blog/2022/02/17/dockershim-faq/) in v1.24, the CRI is the only supported and documented way through which Kubernetes interacts with different container runtimes. Each kubelet negotiates which version of CRI to use with the container runtime on that node.
+
+In the previous release, the Kubernetes project recommended using CRI version `v1`, but kubelet could still negotiate the use of CRI `v1alpha2`, which was deprecated.
+
+Kubernetes v1.26 drops support for CRI `v1alpha2`. That [removal](https://github.com/kubernetes/kubernetes/pull/110618) will result in the kubelet not registering the node if the container runtime doesn't support CRI `v1`. This means that containerd minor version 1.5 and older will not be supported in Kubernetes 1.26; if you use containerd, you will need to upgrade to containerd version 1.6.0 or later **before** you upgrade that node to Kubernetes v1.26. Other container runtimes that only support the `v1alpha2` are equally affected: if that affects you, you should contact the container runtime vendor for advice or check their website for additional instructions in how to move forward.
+
+### Progress on the CSI migration process
+
+Following the GA of the [core Container Storage Interface (CSI) Migration](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/625-csi-migration) feature in the previous release, CSI migration is an on-going effort that we've been working on for a few releases now, and this release continues to add (and remove) features aligned with the migration's goals.
+
+#### Delegate FSGroup to CSI Driver graduated to Stable
+
+This feature allows Kubernetes to [supply the pod's `FSGroup` to the CSI driver when a volume is mounted](https://github.com/kubernetes/enhancements/issues/2317) so that the driver can utilize mount options to control volume permissions. Previously, the kubelet would always apply the `fsGroup`ownership and permission change to files in the volume according to the policy specified  in the `Pod.spec.securityContext.fsGroupChangePolicy` field. Starting with this release, CSI drivers have the option to apply the `fsGroup` settings during attach or mount time of the volumes.
```

Comment:
```suggestion
This feature allows Kubernetes to
[supply the pod's `fsGroup` to the CSI driver when a volume is mounted](https://github.com/kubernetes/enhancements/issues/2317)
so that the driver can utilize mount options to control volume permissions.
Previously, the kubelet would always apply the `fsGroup`ownership and permission change
to files in the volume according to the policy specified  in the Pod's
`.spec.securityContext.fsGroupChangePolicy` field.
Starting with this release, CSI drivers have the option to apply the `fsGroup` settings during
attach or mount time of the volumes.
```